crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -36,6 +36,11 @@ use std::{env, str::FromStr};
 
 const MAX_GAS: u64 = 10000;
 
+// TODO: passing empty list of dependencies to build_and_try_publish_test_package will fail this
```

Comment:
Probably the issue here is that this package still depends on `Sui` and `MoveStdlib` if not `object_basics`.